### PR TITLE
Added Zhiltsov's report for 19/10/'15

### PR DESCRIPTION
**Pull request by zhiltsov-max**

### CODE REVIEW

File: 2015-10/WR-2015-10-19.md [hunk 1]
```diff
@@ -40,7 +42,8 @@
   1. Гладилов Глеб
      -
   1. Жильцов Максим
-     -
+     - Видимо, подключить модуль denseCRF и проверить работоспособность.
+     - Понять, как обучилась сеть. Провести обучение с исправленной моделью. Протестировать.
```
Comment (valentina-kustikova) [inline 2015-10/WR-2015-10-19.md]: @zhiltsov-max, что такое исправленная сеть?

Comment (zhiltsov-max) [inline 2015-10/WR-2015-10-19.md]: В предыдущих запусках не было видно какого-либо обучения, результаты были похожи на случайные. Надо разобраться, в чем же дело, и повторить процесс. Есть варианты:
- смотреть на batch_size и iter_size + добавить код от основной ветки
- смотреть на параметры обучения. Например, в слое fc8 имеется blob_lr: 10, 20, отличающиеся от остальных на порядок. Хотел проверить этот момент, только с cuda-5.5  что-то случилось.

Comment (valentina-kustikova) [inline 2015-10/WR-2015-10-19.md]: Понятно. Очень похоже, что Сергей переставил драйвера cuda с версии 5.5 на 7.0.

Comment (druzhkov-paul) [inline 2015-10/WR-2015-10-19.md]: @zhiltsov-max, увеличенный на порядок локальный learning rate --- это норма для дообучения (fine-tuning), хотя, в общем-то, и не обязательно. Вы хотите, чтобы веса (параметры модели), которые инициализируются случайно, менялись быстрее, чем загруженные из готовой (неслучайной) модели, предполагая, что они уже достаточно хорошие. Для обучения с нуля это особого смысла не имеет.

Я еще раз посмотрел на параметры того эксперимента, где "результаты были похожи на случайные". Вы обучали модель с нуля? Если да, то это может требовать гораздо больше времени, по сравнению с дообучением похожей модели, и на представленном промежутке времени просто ничего и не успело обучиться. Лучше сразу начинать с неслучайной модели, и это надо сделать в первую очередь.

Comment (zhiltsov-max) [inline 2015-10/WR-2015-10-19.md]: @druzhkov-paul, те параметры есть как в просто обучении, так и в дообучении. После того эксперимента я проводил еще один, на 20к и 14к итераций соответственно. Результаты не сильно отличаются, логи остались. Обучение проводилось с нуля. Возможно, надо попытаться взять начальные значения из vgg16, как делали авторы статьи. Еще один момент заключается в том, что обученная модель, выложенная в сеть, называется train2_iter_8000, хоть на этом и не стоит строить большую теорию.
Кроме того, появилась справка по выводу процесса обучения:
Там есть 3 вывода на каждые 10 итераций. Значения означают
- accuracy / pixel accuracy
- recall / class accuracy
- Jaccard / IOU
